src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1162,7 +1199,6 @@ def ZCLSend(self, cluster, command, nodeid, endpoint, groupid, args, blocking=Fa
     def ZCLReadAttribute(self, cluster, attribute, nodeid, endpoint, groupid, blocking=True):
         self.CheckIsActive()
 
-        req = None
         clusterType = eval(f"GeneratedObjects.{cluster}")
```

Comment:
Maybe instead of `eval` we can use ?:
```python
clusterType = getattr(GeneratedObjects, cluster)
```

If such refactor will work, remove noqa from import.